chsdi/tests/integration/test_mapservice.py

```diff
@@ -34,6 +34,14 @@ def test_metadata_chargeable_false(self):
         resp = self.testapp.get('/rest/services/blw/MapServer', params={'chargeable': 'false'}, status=200)
         self.assertEqual(resp.content_type, 'application/json')
 
+    def test_faqlist(self):
+        resp = self.testapp.get('/rest/services/api/faqlist', status=200)
+        self.assertEqual(resp.content_type, 'application/json')
+
+    def test_faqlist_topic_all(self):
+        resp = self.testapp.get('/rest/services/all/faqlist', status=200)
+        self.assertEqual(resp.content_type, 'application/json')
+
```

Comment:
Please check here that we have at least x layers in the results. (x high, but 2-3 smaller than current count).

Also, check for correct structure (by checking for the 4 lists, and 1-2 layers in each list)